src/serializer/serializer.js

```diff
@@ -610,6 +582,10 @@ export class Serializer {
       if (init) {
         if (init !== id) {
           if (target.usedBySingleFunctionValue) {
+            let declar = t.variableDeclaration((bindingType ? bindingType : "var"), [
```

Comment:
We emit a "let" only if the original program used a "let". Otherwise, we stick to "var" as by default we emit ES5 code.